### PR TITLE
fix: Allow for logging URL/URLPattern.prototype

### PR DESCRIPTION
**Pull request by kamilogorek**
Instead of throwing, it now prints:

```
URL {
  href: [Getter/Setter],
  origin: [Getter],
  protocol: [Getter/Setter],
  username: [Getter/Setter],
  password: [Getter/Setter],
  host: [Getter/Setter],
  hostname: [Getter/Setter],
  port: [Getter/Setter],
  pathname: [Getter/Setter],
  hash: [Getter/Setter],
  search: [Getter/Setter
}
```

Fixes https://github.com/denoland/deno/issues/19140
Fixes https://github.com/denoland/deno/issues/19128

### CODE REVIEW

**Comment by crowlKats** [comment]
could we get a test?

**Comment by kamilogorek** [comment]
It looks like there was already a test that tested a part of it, which broke now. Will get back to it later and fix it up.

File: cli/tests/testdata/run/prototype_logging.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+console.log(URL.prototype);
+console.log(URLPattern.prototype);
```
[the remark this answers is not in the document]
Comment (kamilogorek) [reply cli/tests/testdata/run/prototype_logging.ts:2]: No specific reason, I just found `070_location.ts` test which does something very similar, and decided to put it in the same place.

**Review by bartlomieju — CHANGES_REQUESTED** [review]
Marking as "Request changes" because we discovered that the same problem appears for other Web APIs. @crowlKats will look into more general solution next week.

**Pull request closed without merging**